docs/content/guides/developer/app-examples/e2e-counter.mdx

```diff
@@ -1,176 +1,225 @@
 ---
 title: Distributed Counter
+hide_table_of_contents: true
 ---
 
-This example walks you through building a basic distributed counter app, covering the full end-to-end flow connecting your Move code to your client app. The app allows you to create counters that anyone can increment, but only the owner can reset. This example assumes you already have a React App set with dApp Kit, and it's required `Providers` as described in [Client App with Sui TypeScript SDK](../first-app/client-tssdk.mdx).
+This example walks you through building a basic distributed counter app, covering the full end-to-end flow of building your Sui Move module and connecting it your React Sui dApp. The app allows users to create counters that anyone can increment, but only the owner can reset. 
 
-{@include: ../../../snippets/info-pnpm-required.mdx}
+The guide is split into two parts:
 
-If haven't followed Client App with Sui TypeScript SDK, run the following command in a terminal or console to scaffold a new app:
+1. [Smart Contracts](#smart-contracts): The Move code that sets up the `Counter` structure and logic.
+1. [Frontend](#frontend): A UI that enables users to create, increment, and reset `Counter` objects.
 
-```bash
-pnpm create @mysten/dapp --template react-client-dapp
-```
+:::tip Additional resources
 
-or
+[Example source code](https://github.com/MystenLabs/sui/blob/c617f8dced6c61b7c2070a51999bc202d43f7375/examples/distributed-counter)
 
-```bash
-yarn create @mysten/dapp --template react-client-dapp
-```
+:::
 
-To get a head start, you can automatically create this example using the following `template` value instead:
+## What the guide teaches
 
-```bash
-pnpm create @mysten/dapp --template react-e2e-counter
-```
+- **Shared objects:** The guide teaches you how to use [shared objects](../../../concepts/object-ownership/shared.mdx), in this case to create globally accessible `Counter` objects.
 
-or
+## What you need
 
-```bash
-yarn create @mysten/dapp --template react-e2e-counter
-```
+Before getting started, make sure you have:
+
+- [Installed the latest version of Sui](../getting-started/sui-install.mdx).
+
+## Smart contracts {#smart-contracts}
+
+In this part of the guide, you write the Move contracts that create, increment, and reset counters. The first step is to [set up a Move package](../first-app/write-package.mdx) for storing your Move modules.
+
+:::note
```

Comment:
```suggestion
:::note
```
Missed this the first time. I think we want these to be `:::info` instead. The note ones don't really show up too well.

Reply:
Gotcha, will update now. Looking at the [style guide](https://docs.sui.io/style-guide#note), does the :::info box follow similar logic to the purpose of the :::note box?